frontend/src/metabase/components/Triggerable.jsx

```diff
@@ -6,6 +6,9 @@ import { isObscured } from "metabase/lib/dom";
 import Tooltip from "./Tooltip";
 
 import cx from "classnames";
+import styled from "@emotion/styled";
+
+const Trigger = styled.a``;
```

Comment:
`Trigger` using emotion so we can use the `as` property

Reply:
Nice 👏